target_snowflake/__init__.py

```diff
@@ -170,6 +173,21 @@ def persist_lines(config, lines, table_cache=None, file_format_type: FileFormatT
             else:
                 records_to_load[stream][primary_key_string] = o['record']
 
+            if archive_load_files_enabled and stream in archive_load_files_data:
```

Comment:
```suggestion
            if archive_load_files and stream in archive_load_files_data:
```

can we change it to be single level option?